sessionctx/variable/tidb_vars.go

```diff
@@ -373,6 +373,11 @@ const (
 	// Value set to `false` means never use mpp.
 	TiDBAllowMPPExecution = "tidb_allow_mpp"
 
+	// TiDBBanTiFlashCop means we only use MPP mode to query data.
+	// Default value is `true`, means to be determined by the optimizer.
+	// Value set to `false` means we may fall back to TiFlash cop plan if possible.
+	TiDBBanTiFlashCop = "tidb_ban_tiflash_cop"
```

Comment:
Double negative is hard to follow. 
Can we rename to 'tidb_allow_tiflash_cop' or 'tidb_enable_tiflash_cop'?

Reply:
make sense